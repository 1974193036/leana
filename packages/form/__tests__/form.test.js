import ssyyForm from "../src/form.vue"
import { mount } from "@vue/test-utils"

describe('ssyy-form', () => {
  test('form-snapshot', () => {
    const wrapper = mount(ssyyForm)
    expect(wrapper.vm.$el).toMatchSnapshot() // 生成快照
  })
})