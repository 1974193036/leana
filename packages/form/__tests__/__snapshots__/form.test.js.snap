// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ssyy-form form-snapshot 1`] = `
<form
  class="aaa"
/>
`;
